### ISSUE TITLE
transphobic slurs are very funny

### ISSUE THREAD

**Issue by faster-than-reft** [alloca123/YMG-LICENSE#2]
i am laughing, the whole squad, even, is laughing 😐

**Comment by istrictlyusec89** [comment]
i too have an irrational fear of trannies entering the women's bathrooms and [redacted for safe space reasons]

**Comment by alloca123** [comment]
I disagree, Trainsphobic slurs are not a laughing matter. We should be more inclusive of trains and stop saying trainsphobic things like "cars and airplanes are better". Every day trains are being oppressed by society. We need to be more accepting of trains, go back to like it was in the 1800s where people were inclusive and even encouraging of trains. Any trains reading this need to remember, you are worth something and are equal to cars and airplanes. #trainsrights

**Comment by RainbowAsteroids** [comment]
and this guy has his main branch named "master" like what kind of KKK member is @alloca123?

**Comment by alloca123** [comment]
@RainbowAsteroids yeah I am actually the leader of the kool kidz klub, I'm also known as kalvin koolage(i am so famous that i have my own [wikipedia article](https://en.wikipedia.org/wiki/Calvin_Coolidge)). We at the kool kidz klub welcome you with open arms.

![here is a picture if you don't believe me](https://0x0.st/-bMX.jpeg)

**Comment by RainbowAsteroids** [comment]
Look mr grand wizard, that sounds pretty cool, but I've got an issue with your branch's name, and before you put me on a blacklist, I'd like to say that it would be pretty smart if you change the name of the branch from `master` to `mstr`, so your project can have a hip and cool silicon vally vibe to attract users.

**Comment by advaithm** [comment]
Hi, gender fluid person here. I don't know why you decided to keep this slur in your license it hurts me when I see people mocking my very existence for a few laughs. I understand the point of this license but seriously can't come up with something more original then a slur. please stop insulting people who have done nothing to insult you

**Comment by alloca123** [comment]
@advaithm What is gender fluid? Am i able to drink that? Why are you made of pure liquid instead of bits of solid? how does it feel to be a liquid, you mentioned the pain that comes with being a liquid and not having any bones, what other struggles do you deal with as a result of being a liquid? how am i insulting you?

**Comment by advaithm** [comment]
Gender fluid means that I am not one gender all the time. Please change your license

**Comment by istrictlyusec89** [comment]
why don't you change your mentall illness, bigot? how dare you try to impose on this nice xer's license's life.

LICENSE LIVES MATTERS!

**Comment by alloca123** [comment]
@advaithm I don't know what license changes you are requesting. And how is a puddle of fluid supposed to have a gender? Btw how are you messaging me right now? i can imagine that puddles of fluid have issues using a keyboard.

**Comment by alloca123** [comment]
@advaithm alright, but you have claimed to be a furry and all your opinions no longer matter as a result.

**Comment by Architector4** [comment]
Don't feed the trolls, just report.

**Comment by alloca123** [comment]
@Architector4 who are you reporting and why

**Comment by istrictlyusec89** [comment]
@Architector4 thank you for the tip. i have reported that filthy troll and have awarded you with the highest honors in the lands

**Comment by alloca123** [comment]
@Architector4 oh are you talking about the trainsphobe and the furry? So they were just pranking me? Wow i should've known that people made out of "gender fluid" don't exist. One can not survive as just a puddle of fluid, yet they tricked me into thinking these people actually existed. I know humans are largely made up of water, but people made up 100% of this gender fluid don't exist. They probably couldn't put on a fursuit if they are just a puddle of fluid. But you don't need to report them, they were just having fun(and managed to trick me into believing that people made out of pure fluid exist).
